tests/lib/rules/no-unused-prop-types.js

```diff
@@ -784,54 +764,6 @@ ruleTester.run('no-unused-prop-types', rule, {
       ].join('\n')
     }, {
       code: [
-        'const statelessComponent = (props) => {',
```

Comment:
It's not the same test; I see 4 different variations: arrow, non-arrow, and destructured, and non-destructured.